reference/commands/profile.rst

```diff
@@ -26,7 +26,7 @@ conan profile
 
 
 conan profile detect
-====================
```

Comment:
@czoido should we update `develop2` with `release/2.0.0-beta`?

Reply:
Should be updated now, but please point all the PR that have changes not related specifically with beta10 to `release/2.0.0-beta` branch